fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/ResourceFactory.java

```diff
@@ -28,6 +28,16 @@
  */
 public interface ResourceFactory {
 
+    /**
+     * Get a resource from the persistence layer without using a transaction.
+     *
+     * @param identifier The path or identifier for the resource.
+     * @return The resource.
+     * @throws PathNotFoundException If the identifier cannot be found.
+     */
+    public FedoraResource getResource(final String identifier)
```

Comment:
I believe you also need an _Instant version_ param, since we'll need this call for getting mementos. 

You could also just add an overloaded version of the above method with the added _Instant version_ param.   So I'm suggesting something like this: 

public FedoraResource getResource(final String identifier)
public FedoraResource getResource(final String identifier, Instant version)

Reply:
above should be a change request.